### PR TITLE
Edited "What is surf" and added "How surf works".

### PR DESCRIPTION
**Pull request by Jedno**

### CODE REVIEW

File: _posts/guides/2021-12-21-guide_gamemode-info-surf.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ toc_sticky: true
 
 ## What is Surf?
 
-Surf is a very popular gamemode
+Surf is a very popular gamemode that derives from the same source bug as bhop and ahop. Even thought it came from a bug it became one of the most popular minigames on games like cs:s cs:go team fortress 2 etc.
```
Comment (tsa96) [inline _posts/guides/2021-12-21-guide_gamemode-info-surf.md:20]: I'm not sure it's right to call surf movement a "bug" (and it's certainly not the same as ahop), it's a product of increasing sv_airaccelerate significantly to give you greater air control, the same as with bhop and I believe most climb variants. I also wouldn't call it a minigame, I think gamemode/sub-game/custom server might be better.

Typo in "Even thought".

Let's try and improve our punctuation a little, "Even thought it came from a bug it became" can be "Even thought it came from a bug, it became" (though as above, I don't think it's right to call it a bug).

The game names should be capitalised.

File: _posts/guides/2021-12-21-guide_gamemode-info-surf.md [hunk 2]
```diff
@@ -27,4 +27,7 @@ Surf is a very popular gamemode
 
 ## How Surf Works
 
-TODO
\ No newline at end of file
+You can surf by holding A on the right side of the ramp and D on the left without holding W.
```
Comment (tsa96) [inline _posts/guides/2021-12-21-guide_gamemode-info-surf.md:30]: Let's change "You can surf" to something like "Surfing on a ramp requries". Comma between "left" and "without".

File: _posts/guides/2021-12-21-guide_gamemode-info-surf.md [hunk 3]
```diff
@@ -27,4 +27,7 @@ Surf is a very popular gamemode
...
+When getting on a ramp you need to look downwards, then slowly turn your camera until the bottom where you should be looking flat with the ramp.
+If you want to jump you need to slowly start looking at the ramp and when you're halfway up look directly at it.
```
Comment (tsa96) [inline _posts/guides/2021-12-21-guide_gamemode-info-surf.md:32]: I'm a little confused by this, do you mean when starting a stage? If that's the case let's clarify with by starting with "To begin boarding the ramp by jumping". Though isn't this also the case if you don't jump?

File: _posts/guides/2021-12-21-guide_gamemode-info-surf.md [hunk 4]
```diff
@@ -27,4 +27,7 @@ Surf is a very popular gamemode
...
+When you are mid-air its beneficial to strafe by looking left while holding A then switcing to looking right while holding D and repeating that.
```
Comment (tsa96) [inline _posts/guides/2021-12-21-guide_gamemode-info-surf.md:33]: Typo on "switching". I don't like the "and repeating that", I think you're talking about strafing for speedgame somewhat but that's not necessary, the point should be about always holding the correct strafe key when turning.

**Review by tsa96 — CHANGES_REQUESTED** [review]
This has some potential but needs a bit of work.

Firstly there's a handful of typos and punctuation errors, I'd suggest running it through a spell checker like on Google Docs at the very least.

**Comment by antjlo** [comment]
> This has some potential but needs a bit of work.
> 
> Firstly there's a handful of typos and punctuation errors, I'd suggest running it through a spell checker like on Google Docs at the very least.

Easybib and grammarly are free I don't know any others but you can use them in conjuction if you want it as perfect as possible back and fourth, its annoying but i've done it before and it worked

File: _posts/guides/2021-12-21-guide_gamemode-info-surf.md (hunk 1 above)
Comment (tsa96) [reply _posts/guides/2021-12-21-guide_gamemode-info-surf.md]: @Jedno last change is a little better, but please capitalise the game names properly.

Also, lets add a little bit to explain why air control is important, perhaps: _increasing sv_airaccelerate setting, giving better air control thereby allowing the player to perform sharp turns in the air. This also allows the player to "surf" along angled surfaces known as "ramps"._

File: _posts/guides/2021-12-21-guide_gamemode-info-surf.md (hunk 4 above)
Comment (tsa96) [reply _posts/guides/2021-12-21-guide_gamemode-info-surf.md]: Still not too happy with this, still the issue in the latter part of my above comment.

**Comment by tsa96** [comment]
Okay I've spoken with the team on this and for this to get merged we'd really need to see something more fleshed out.

Our goal for these pages is to make something that comprehensively explains the mode, including much more detailed descriptions of what the gamemode is and how the mechanics work (a good example is the [ahop section](https://docs.momentum-mod.org/guide/gamemode-ahop/)). I'm afraid this PR feels too small to really even help build off of in that regard. I've been trying to keep with some suggestions in comments here but I think if we are going to build this into something worth merging, we should discuss it more in Discord rather than in Github review.

So: if you wanna to keep working on this then great, but lets move over to Discord and work on it there to try make something more substantial, maybe with input from some others there, I'll be happy to help.

I'm going to close this PR for now, lets work on it some more, and if we can develop this into something more substantial you can open a new one.

**Pull request closed without merging**